llvm/include/llvm-c/Core.h

```diff
@@ -4611,6 +4629,22 @@ int LLVMGetMaskValue(LLVMValueRef ShuffleVectorInst, unsigned Elt);
 LLVMBool LLVMIsAtomicSingleThread(LLVMValueRef AtomicInst);
 void LLVMSetAtomicSingleThread(LLVMValueRef AtomicInst, LLVMBool SingleThread);
 
+/**
+ * Returns whether an instruction is an atomic instruction, e.g., atomicrmw,
+ * cmpxchg, fence, or loads and stores with atomic ordering.
+ */
+LLVMBool LLVMIsAtomic(LLVMValueRef Inst);
```

Comment:
Maybe this should be `LLVMInstructionIsAtomic`? The API isn't consequent on this.

Reply:
Looks like we currently don#t have any functions named `LLVMInstructionIs*`, so I think this is ok.